beginner_bioinf_tools.py

```diff
@@ -0,0 +1,112 @@
+def run_dna_rna_tools(inputs: tuple) -> list or str:
+    """
+    Produce a list of either transcripts, reverse sequences, complementary sequences or reverse complementary sequences
+    arguments:
+    - inputs (tuple): an orbitrary amount of strings where the last one is the name of desired operation, and other strings are sequences
+    return
+    - complement_seqs (list): a list of complementary sequences
+    """
+    from modules.dna_rna_tools import check_input, transcribe, reverse, complement, reverse_complement
+    if len(inputs) < 2:
+        raise ValueError('Invalid input: the function requires at least one sequence and an operation name!')
+    *seqs, operation = inputs
+    check_input(seqs, operation)
+    if operation == 'transcribe':
+        result = transcribe(seqs)
```

Comment:
```suggestion
        results = transcribe(seqs)
```
Так будет понятно что это коллекция:)